.travis.yml

```diff
@@ -70,13 +70,14 @@ before_script:
   - if [[ $BEHAT_TEST == cms ]]; then composer require silverstripe/recipe-cms:4.3.x-dev --no-update; fi
 
   - composer install --prefer-source --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+  - composer show
```

Comment:
Is the output from `composer install` not sufficient?

Reply:
well, technically it is, but `show` gives it ordered by alphabet and easier to read by humans (IMHO)